### PR TITLE
Feat: Updated Site Styles

### PR DESCRIPTION
**Pull request by MykalMachon**
## Summary of PR
This pr should outline some basic style changes to the site to improve general readability and get us on a path to a more attractive site in general. Here's a list of a few thing's I'll be trying to implement on this branch: 

### Goals
- [ ] Improved mobile readability
- [ ] Improve fonts & font structure across the site
- [ ] Improve visual hierarchy of document (h1, h2, h3 should be distinct) 
- [ ] General cleanup to create something a bit more modern

### CODE REVIEW

**Comment by MykalMachon** [comment]
- [x] Improving mobile readbility
- [x] Improving visual hierarchy

See the following screenshots of the new font setup on the site. 

**Old / Current View**
old_mobile
old_desktop

**New View**
new_desktop
new_mobile

This should be a significant improvement overall

File: src/styles/main.css [hunk 1]
```diff
@@ -88,8 +90,6 @@ main {
   padding-right: 32px;
 
   background-color: #d3d3d3;
-
-  animation: slide-in 500ms ease-in;
```
Comment (CalebLangbroek) [inline src/styles/main.css:92]: Did you want to remove the animations?

**Review by CalebLangbroek — COMMENTED** [review]
Looks good to me. The whole site could definitely use a CSS makeover though.

**Comment by MJJBennett** [comment]
Looks good to me, although imo whitespace should be fully removed on mobile layouts; that's just personal preference, though. Would be nice if we could zoom out the header image a bit; not to fit it all in, but at least the full name of the CSA fitting would be nice (& it would be less pixelated, ideally)

File: src/styles/main.css (hunk 1 above)
Comment (MykalMachon) [reply src/styles/main.css:92]: Yup! I did. 
On mobile it was pretty jarring- My branch has kind of turned into a bit of a more holistic makeover 😅
I'm going to be pushing some things up either tonight or tomorrow that redo the nav, add a footer, and cleanup the general page templates.

**Comment by MykalMachon** [comment]
> Looks good to me, although imo whitespace should be fully removed on mobile layouts; that's just personal preference, though. Would be nice if we could zoom out the header image a bit; not to fit it all in, but at least the full name of the CSA fitting would be nice (& it would be less pixelated, ideally)

Agreed. on both of these points. 
I'll have some more commits to address this in the next few days

**Comment by MykalMachon** [comment]
I'm going to be closing this as we're going to be starting from scratch on a new site.

**Pull request closed without merging**